notebooks/CubevizExample.ipynb

```diff
@@ -27,13 +64,117 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from astropy.utils.data import download_file\n",
-    "\n",
-    "# This file is originally from https://data.sdss.org/sas/dr14/manga/spectro/redux/v2_1_2/7495/stack/manga-7495-12704-LOGCUBE.fits.gz\n",
-    "# but has been modified to correct some inconsistencies in the way units are parsed\n",
     "fn = download_file('https://stsci.box.com/shared/static/28a88k1qfipo4yxc4p4d40v4axtlal8y.fits', cache=True)\n",
-    "viz.load_data(fn)"
+    "\n",
+    "with warnings.catch_warnings():\n",
+    "    warnings.simplefilter('ignore')\n",
+    "    cubeviz.load_data(fn)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The upper left viewer containing the FLUX cube is the default viewer that is accessible as follows."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "viewer = cubeviz.default_viewer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note also that in the above example there are mouse-over coordinates visible by default."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It possible to make selections/regions in images and export these to astropy regions. Click on an image viewer toolbar then click on the circular selection tool, and drag and click to select an interesting region on the sky. We can then export this region with:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "regions = cubeviz.get_interactive_regions()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "regions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It is also possible to programmatically pass a `regions` shape, a `photutils` aperture shape, or a Numpy mask into Cubeviz.\n",
+    "\n",
+    "*NOTE: Sky regions are not yet supported for data cubes, so the sky regions here will not load successfully.*"
```

Comment:
See https://github.com/glue-viz/glue-astronomy/issues/75

Reply:
If we decide to remove the sky regions from the notebook, this note could be adapted to say that they aren't currently supported but will be at some point.

Reply:
I am not promising anything. I removed the second half of the note though. Thanks!